SQL1/Homework1-SQL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE ZooManagement5;
+
+USE ZooManagement5;
+
+CREATE TABLE Animals1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfAnimal VARCHAR(30),
+    NumberOfAnimal INT,
+);
+
+CREATE TABLE Species1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfSpecies VARCHAR(20),
+    NumberOfSpecies INT,
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE CareTakers1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    FirstName VARCHAR(30),
+    LastName VARCHAR(30),
+    AreaOfWork VARCHAR(20),
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE ZooAreas1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfTheArea VARCHAR(20),
+    CareTakersId INT REFERENCES CareTakers1(Id),
+);
+
+CREATE TABLE Visitors1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    DayOfTheWeek VARCHAR(30),
+    NumbersOfVisitors INT,
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE Tickets1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Validity VARCHAR(30),

```

Comment:
For Validity a new table, maybe called, ValidityTypes could be used. Inside ValidityTypes we can store information like "one day", "one month", "one week" etc. and then it can be referenced in the Tickets table.